bdk-android/plugins/src/main/kotlin/org/bitcoindevkit/plugins/UniFfiAndroidPlugin.kt

```diff
@@ -26,11 +26,9 @@ internal class UniFfiAndroidPlugin : Plugin<Project> {
             executable("cargo")
             args(cargoArgs)
 
-            // if ANDROID_NDK_ROOT is not set then set it to github actions default
+            // if ANDROID_NDK_ROOT is not set, stop build
             if (System.getenv("ANDROID_NDK_ROOT") == null) {
-                environment(
-                    Pair("ANDROID_NDK_ROOT", "${System.getenv("ANDROID_SDK_ROOT")}/ndk-bundle")
-                )
+                throw IllegalStateException("ANDROID_NDK_ROOT environment variable is not set; cannot build library")
```

Comment:
Interesting, good to know how you stop the build here

Reply:
Yeah and I can actually check for it a bit earlier and remove duplicated code. Pushed a fix just now.